tutorials/ROC.ipynb

```diff
@@ -837,14 +838,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "fcst = beta.rvs(2, 1, size=1000)\n",
+    "fcst = beta.rvs(0, 1, size=1000)\n",
```

Comment:
@BethEbert , can you revert this code back? The code was correct, but the description above was wrong (as you correctly picked up). The first arg in `beta.rvs(a, ...)` is a shape parameter in the Beta distribution (see examples of the impact of the shape parameters [here]( https://en.wikipedia.org/wiki/Beta_distribution#/media/File:Beta_distribution_pdf.svg)) and isn't related to the typo that you picked up on above. It becomes undefined when that first arg is 0, which causes the code to break.

Reply:
Great - the updates are now passing after that update